src/control/content/components/LocationForm.js

```diff
@@ -278,6 +285,17 @@ class LocationForm extends React.Component {
             className='form-control' />
         </div>
 
+        <div className='form-group'>
+          <label htmlFor='deeplink'>Query String URL</label>
+          <input
+            disabled            
+            maxLength={ 90 }
+            value={ deeplinkUrl }
+            name='deeplink'
+            className='form-control'
+            type='text'/>
+        </div>
```

Comment:
Add copy link button

Reply:
Instead of adding here, copy link functionality is moved on the list view, as a quick action button.